nodeletctl/pkg/nodeletctl/consts.go

```diff
@@ -15,6 +15,7 @@ const (
 	NodeConverged      = "converging"
 	NodeHealthy        = "ok"
 	CACertExpiryYears  = 3
+	CAExpiryThreshold  = 90
```

Comment:
Is this days/ hours etc? can you update the variable name to reflect the unit?

Reply:
Yes will do